wordpress-mobile/WordPressKit-iOS
# Integrates WP Shared 1.8.13-beta.

## Description

Integrates WP Shared 1.8.13-beta into WPKit.  This is necessary to resolve the issue linked below.

## Associated PRs and issues:

Related to: https://github.com/wordpress-mobile/WordPress-iOS/issues/13154
Relevant integrated WP-Shared changes: https://github.com/wordpress-mobile/WordPress-iOS-Shared/pull/244

## Testing Details

1. Check out WPiOS branch `issue/13154-remove-gallery-captions-from-excerpts`.
2. Install the pods and make sure all is fine.
3. Test the issue reported [here](https://github.com/wordpress-mobile/WordPress-iOS/issues/13154) and make sure it's no longer a problem.

## Checklist:
